core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/components/dot-edit-content-sidebar-section/dot-edit-content-sidebar-section.component.ts

```diff
@@ -0,0 +1,32 @@
+import { CommonModule } from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ContentChild,
+    TemplateRef,
+    input
+} from '@angular/core';
+
+/**
+ *  Component that renders a section with a title and an optional action template.
+ */
+@Component({
+    selector: 'dot-edit-content-sidebar-section',
+    standalone: true,
+    imports: [CommonModule],
+    templateUrl: './dot-edit-content-sidebar-section.component.html',
+    styleUrl: './dot-edit-content-sidebar-section.component.scss',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotEditContentSidebarSectionComponent {
+    /**
+     * The title of the section.
+     */
+    $title = input.required<string>({ alias: 'title' });
+
+    /**
+     * The action template for the section.
+     */
+    @ContentChild('sectionAction')
+    actionTemplate: TemplateRef<unknown>;
```

Comment:
Maybe use contentChild based on signals?

Reply:
Not work as expected.